### PR TITLE
Navigation flattening

### PR DESCRIPTION
**Pull request by jurgenbelien**
Removed horizontal list of books, horizontal list of pages in books. Collected horizontal list thingies in Bottom Shelf component

### CODE REVIEW

File: components/card-component/CardComponent.vue [hunk 1]
```diff
@@ -15,12 +17,25 @@ export default {
   components: {
     SvgIcon
   },
-  props: [ 'body', 'count', 'image', 'path', 'slug', 'theme', 'title', 'video' ]
+  props: [ 'body', 'count', 'delay', 'image', 'path', 'slug', 'theme', 'title', 'video' ],
```
Comment (celinekurpershoek) [inline components/card-component/CardComponent.vue]: Video and image aren't used

File: components/bottom-shelf/BottomShelf.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <transition name="slide-up">
+    <div class="bottom-shelf">
+      <slot>
+        <!-- content -->
+      </slot>
+    </div>
+  </transition>
+</template>
+
+<script>
```
Comment (celinekurpershoek) [inline components/bottom-shelf/BottomShelf.vue]: Maybe remove empty script tag?

File: components/card-list/CardList.vue [hunk 3]
```diff
@@ -15,33 +16,36 @@
 </template>
 <script>
 import CardComponent from '~/components/card-component/CardComponent'
+
 export default {
   components: {
     CardComponent
   },
-  props: [ 'cards' ]
+  props: [ 'book', 'chapter', 'cards' ]
```
Comment (celinekurpershoek) [inline components/card-list/CardList.vue]: Book and chapter aren't used

File: pages/narratives/_book/index.vue [hunk 4]
```diff
@@ -1,33 +1,23 @@
...
-  <div>
-    <div class="list--scroll">
-      <h1>{{title}}</h1>
-      <title-list v-bind:titles="books" v-bind:active="[{ slug }]" v-bind:exclude="true" />
-    </div>
-    <card-list v-bind:cards="chapters" />
-  </div>
+  <bottom-shelf>
+    <narrative-header v-bind:book="{ title, slug, path}" />
+    <card-list v-bind:cards="chapters" v-bind:book="{ title, slug, path}"/>
```
Comment (celinekurpershoek) [inline pages/narratives/_book/index.vue]: Book isn't used in the card-list component

File: pages/narratives/_book/index.vue [hunk 5]
```diff
@@ -1,33 +1,23 @@
...
+  </bottom-shelf>
 </template>
 
 <script>
+import BottomShelf from '~/components/bottom-shelf/BottomShelf'
 import CardList from '~/components/card-list/CardList'
-import TitleList from '~/components/title-list/TitleList'
-import books from '~/static/data/books/index.json'
 import events from '~/components/events/events'
+import NarrativeHeader from '~/components/narrative-header/NarrativeHeader'
```
Comment (celinekurpershoek) [inline pages/narratives/_book/index.vue]: Do we still need those events?

File: pages/narratives/_book/_chapter/index.vue [hunk 6]
```diff
@@ -1,37 +1,106 @@
...
+import PageComponent from '~/components/page-component/PageComponent'
 import events from '~/components/events/events'
+import loadData from '~/lib/load-data'
 
 export default {
   async asyncData (context) {
-    const { location, pages, path, slug, title } = await loadData(context, context.params)
-    return { location, pages, path, slug, title }
+    const { pages } = await loadData(context, context.params)
+    const book = pages[0].book
```
Comment (celinekurpershoek) [inline pages/narratives/_book/_chapter/index.vue]: Maybe write this one as:

```
const page = pages[0]
return {
book: page.book,
chapter: page.chapter,
pages,
slug: context.params.page,
}

File: components/narrative-header/NarrativeHeader.vue [hunk 7]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <header v-if="chapter" class="narrative-header">
+      <!-- Book -->
```
Comment (celinekurpershoek) [inline components/narrative-header/NarrativeHeader.vue]: `<!-- book -->` Leftover comment?

Comment (jurgenbelien) [reply components/narrative-header/NarrativeHeader.vue]: more like missing `<!-- chapter -->` comment further down :P

File: components/card-component/CardComponent.vue (hunk 1 above)
Comment (jurgenbelien) [reply components/card-component/CardComponent.vue]: I'll change this to a more orderly list of props!

File: pages/narratives/_book/index.vue (hunk 5 above)
Comment (jurgenbelien) [reply pages/narratives/_book/index.vue]: yes, line 27 to 30.

Comment (celinekurpershoek) [reply pages/narratives/_book/index.vue]: collapsed view ;)

**Pull request merged**